src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else if (a.toString() < b.toString()) {
+        return -1;
+      } else {
+        return 0;
+      }
```

Comment:
Consider caching `a.toString()` and `b.toString()` before the if-else block to avoid calling `toString` method multiple times for the same values. This can improve performance especially for large arrays.